src/main/java/seedu/duke/Ui.java

```diff
@@ -80,6 +84,31 @@ public void showClientDeletedConfirmationMessage(Client deletedClient) {
         showToUser("  " + deletedClient);
     }
 
+    public void showPairedPropertiesDeletedConfirmationMessage(Client deletedClient, PairingList pairingList) {
+        HashMap<Client, Property> clientPropertyPairs = pairingList.getClientPropertyPairs();
+        if (clientPropertyPairs.containsKey(deletedClient)) {
```

Comment:
Would be good to assign a boolean variable to  `clientPropertyPairs.containsKey(deletedClient)` so that it's more readable for the reader